frontend/views/components/modal/Modal.vue

```diff
@@ -120,8 +120,9 @@ export default {
         this.subcontent.pop()
       } else {
         this.content = null
-        // Refocus on button that open the modal
-        this.lastFocus.focus()
+        // Refocus on button that open the modal if it exists.
```

Comment:
It's either the element is on the page and we can focus again or it is not, then we don't need to focus back on this element